src/table/boot.rs

```diff
@@ -584,6 +633,7 @@ impl BootServices {
     /// use. Use [`open_protocol`] instead.
```

Comment:
Let's say ``Use [`open_protocol_exclusive`] instead``. The plain `open_protocol` function turned out not to be so safe either :)

Reply:
Done -- updated docs in a few more places and also updated the `deprecated` message on `handle_protocol` and `locate_protocol` to mention `open_protocol_exclusive`